pages/k8s/1.22/release-notes.md

```diff
@@ -114,6 +114,19 @@ This happens when the version of `snapd` on the host does not match the version
 container. As a workaround, ensure the same version of `snapd` is installed on the host and
 in LXD containers.
 
+- [LP 1936816](https://bugs.launchpad.net/bugs/1936816) and [LP 1913228](https://bugs.launchpad.net/bugs/1913228) Filesystem Hierachy Standards
+  
+  Applications running inside a kubernetes-master should set pid files and log files in 
+appropriate operational locations like `/run/` and `/var/log/cdk/`. Care was taken to restart 
+services using these new locations and migrate some existing files out of `/root/cdk/`.
+
+  For the service `cdk.master.auth-webhook` the new pid file and log files are named 
+`/run/cdk.master.auth-webhook.pid` and `/var/log/cdk/cdk.master.auth-webhook.log` 
+to match the systemctl service name.
+
+  If the [`filebeat`](https://charmhub.io/filebeat) charm is related to kubernetes-master,
+ensure that its logpath include this new path ( e.g. `juju config filebeat logpath='/var/log/cdk/*.log'` )
```

Comment:
I wonder since we're making this change if we should drop `cdk` in favor of `ck`.  `ck` is awfully short and could confuse people wondering *what's this `ck` subdirectory for?* so perhaps we should consider `charmed-kubernetes` as our subdir name and `ck` in filenames?